bigtable-hbase-2.x-parent/bigtable-hbase-2.x/src/main/java/com/google/cloud/bigtable/hbase2_x/BigtableAdmin.java

```diff
@@ -145,37 +145,44 @@ public void createTable(TableDescriptor desc, byte[][] splitKeys) throws IOExcep
     List<SnapshotDescription> response = new ArrayList<>();
     for (Snapshot snapshot : snapshotList.getSnapshotsList()) {
       response.add(new SnapshotDescription(
-          snapshot.getName(), 
+          snapshot.getName(),
           TableName.valueOf(snapshot.getSourceTable().getName())));
     }
     return response;
   }
 
-
   /**
    * {@inheritDoc}
-   * 
+   *
    * Calling {@link #addColumn(TableName, ColumnFamilyDescriptor)} was causing stackoverflow.
    * Copying the same code here. //TODO - need to find a better way
    */
   @Override
   public void addColumnFamily(TableName tableName, ColumnFamilyDescriptor columnFamilyDesc)
       throws IOException {
-    modifyColumns(tableName, columnFamilyDesc.getNameAsString(), "add",
-        ModifyTableBuilder.create().add(TableAdapter2x.toHColumnDescriptor(columnFamilyDesc)));
+    ModifyColumnFamiliesRequest request =
```

Comment:
This sequence is very verbose.  Let's discuss an HBase wrapper of ModifyColumnFamiliesRequest

Reply:
Should that wrapper be similar to [ModifyTableAdapter](https://github.com/GoogleCloudPlatform/cloud-bigtable-client/blob/b78bb1519ea4c28fea47dd98555790be046433f8/bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/util/ModifyTableBuilder.java#L81), which may return new `v2.models. ModifyColumnFamiliesRequest` instead of any builder or wrapper?

Reply:
The wrapper should be similar to [`CheckAndMutateUtil.RequestWrapper`](https://github.com/GoogleCloudPlatform/cloud-bigtable-client/blob/master/bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/adapters/CheckAndMutateUtil.java#L84).  The Adapter has a clear "from" and "to" with a single method.  We need something more like a Builder.